Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -1071,6 +1079,9 @@ - (void)setUpInteraction
     _touchActionGestureRecognizer = adoptNS([[WKTouchActionGestureRecognizer alloc] initWithTouchActionDelegate:self]);
     [self addGestureRecognizer:_touchActionGestureRecognizer.get()];
 
+    // FIXME: This should be called when we get notified that loading has completed.
+    [self setUpTextSelectionAssistant];
```

Comment:
I couldn't observe any issue moving this call up 13 lines in this method, as far as I can tell it just initializes or resets the UIWKTextInteractionAssistant